src/button/template/miscComponent.jsx

```diff
@@ -94,7 +94,7 @@ export function Tagline(tagColor : string, impression : ?string, text : string |
         ` } />);
     nodes[1] = (
         <div class={ `${ CLASS.TAGLINE } ${ CLASS.TAGLINE_COLOR }-${ tagColor }` }>
-            <span optional>{ text }</span>
```

Comment:
Why remove `optional`? We still do need to hide the tagline where there isn't space for it right?

Reply:
The text that gets past is `<span optional>{the_tagline}</span`